lib/internal/Magento/Framework/Lock/Backend/Cache.php

```diff
@@ -45,7 +45,7 @@ public function lock(string $name, int $timeout = -1): bool
      */
     public function unlock(string $name): bool
     {
-        return $this->cache->remove($this->getIdentifier($name));
+        return (bool)$this->cache->remove($this->getIdentifier($name));
```

Comment:
Could I kindly ask you to cover this part using a simple unit test? Just mock a behavior that caused an error previously for `$this->cache->remove($this->getIdentifier($name))` and make sure that there's no error after the proposed change.

Thank you!

Reply:
Hello @rogyar 
Tests have already created.
Thank you @engcom-Echo